src/screens/Customers/components/MeterInfoContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { createStyles, withStyles, Theme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { CardPropsInfo } from '../../../components/OutlinedCardList';
+import IconButton from '@material-ui/core/IconButton';
+import { Create as CreateIcon } from '@material-ui/icons';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      width: '50%',
+      display: 'flex',
+    },
+    info: {
+      width: '100%',
+      marginBottom: '8px',
+      padding: '0px 12px 4px',
+      border: '1px solid',
+      height: '60px',
+      maxHeight: '60px',
+      borderRadius: '6px',
+    },
+    icon: {
+      fontSize: '14px',
+      marginLeft: '5px',
+      marginTop: '-4px',
+      color: theme.palette.primary.main,
+    },
+    inline: {
+      display: 'inline-flex',
+    },
+  });
+
+interface MeterInfoContainerProps {
+  classes: { root: string; info: string; icon: string; inline: string; };
+  floatRight: boolean;
+  info: CardPropsInfo;
+  editable?: boolean;
+  gray?: boolean;
+}
+
+function MeterInfoContainer(props: MeterInfoContainerProps) {
```

Comment:
jk I saw in the designs, there is a version of `OutlinedCardList` with an edit button?? Do we know if this is an actual use case?
![Screen Shot 2021-04-09 at 1 20 45 PM](https://user-images.githubusercontent.com/21160510/114236684-d2de4d80-9936-11eb-9457-4927fb9ad4c7.png)

Reply:
I don't know the case for editing a payment (actually, we might not want to make payments editable on the app. I'll think more about it and ask Ashley). The use case for editing a meter reading is if they inputted it wrong (since there's only supposed to be 1 meter reading per period) and need to edit it